react-charts/bullet-chart/step2.md

```diff
@@ -1,24 +1,43 @@
-We'll begin by adding a CSS selector to define the chart size...
-
-- Katacoda is setting up a new React application for you. You'll be ready to code once the server starts and you can see "Welcome to PatternFly" on the lower pane.
-
-## Task
-
-1) Navigate to the `src` folder and open `src/app.css`{{open}}
-
-2) Next we'll add some CSS to define the overall chart size
-
-Copy the following code into the app.css file, replacing all of the content there:
-
-<pre class="file" data-filename="src/app.css" data-target="replace">
-.chart-container {
-  height: 150px;
-  width: 600px;
+Create a simple bullet chart.
+
+1. <strong>Navigate to the `src` folder and open `src/App.js`{{open}}</strong>
+
+2. <strong>Set up the general structure of the chart and the necessary import statements.</strong>
+
+Copy the following code into the App.js file, replacing all of the content there:
+
+<pre class="file" data-filename="App.js" data-target="replace">
+import React from 'react';
+import "@patternfly/react-core/dist/styles/base.css";
+import './app.css';
+import {
+  Chart,
+  ChartBullet,
+  ChartThemeColor
+} from '@patternfly/react-charts';
+
+function App() {
+  return (
+    &lt;div className=&quot;chart-container&quot;&gt;
+      &lt;ChartBullet
+        primarySegmentedMeasureData={[{ name: &#39;Measure&#39;, y: 60 }]}
```

Comment:
Ideally, all of these props (or atleast the main ones) are split up into individual steps and you explain what each prop does at each step

Reply:
Resolved